cli/app.go

```diff
@@ -1496,6 +1497,11 @@ After creation, use 'viam module update' to push your new module to app.viam.com
 					Name:  "generate",
 					Usage: "generate a new modular resource via prompts",
 					Flags: []cli.Flag{
+						&cli.StringFlag{
+							Name:  moduleFlagLanguage,
+							Usage: "resource type to use in module",
+							Value: "python",
```

Comment:
why do we have python here? I might be missing some context but is it acting as a default language?

Reply:
Yep, that's the default language if none is provided